portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I have never been on an airplane'];
```

Comment:
It's better to add more values so you can actually test if it returns different greetings.

Reply:
thanks!